docs/Guides/Customer Voice/SendReviewRequest.md

```diff
@@ -0,0 +1,104 @@
+---
+tags: [Review Request, Customer Voice]
+---
+# Request a Review for a Business Location
+
+With Customer Voice, you can gather authentic reviews via email or text on the sites that matter most, to grow customer loyalty and boost sales.
+
+## Setup
+Create an access token with `customervoice` scopes following the [Authorization guide](../../Authorization/Authorization.md).
+
+Have an activated Customer Voice product and SMS add-on if you need to send SMS review requests. Ensure your email and SMS templates are configured appropriately and your preferred review sites are configured within the Customer Voice platform.
+
+## Step 1: Obtain an Internal Customer ID
+In order to track the status of a review request within Customer Voice, an internal customer record must exist and the id of this record must be provided.
+
+## Step 2: Obtain a Template ID
+Templates control the text and layout of your review request and can be configured to dynamically populate fields from your business information, a customer record or your preferred review sites. 
+
+To obtain a template id, navigate to the "Templates" tab and select "Edit Template" from the configuration menu of your chosen template. 
+![Templates list page](./images/edit-template.png)
+
+Scrolling to the bottom of the "Edit Template" page you will see a button for copying the template ID.
+![Edit template page](./images/copy-template-id.png)
+
+When sending a review request you can also provide a "fallback" template to use in the case where we are unable to send a review request with the primary template. This may occur if, for example, sending the review request would cause you to exceed your monthly SMS limit. If an email fallback template is provided we would send an email review request instead.
+
+## Step 3: Create a Review Request
+<!--
+type: tab
+title: Request
+-->
+Using your generated access token in your `Authorization` header, the internal customer ID and template IDs you can make a `POST` request to send a review request. Notice that the ids provided in the request body contain several ids joined together with ":". For the customer id add the [business ID](../Accounts.md) of the business location you are sending the review request for to the start of your internal customer id. For the template id add the business ID as well as the template type (either "email" or "sms") to the beginning of your template id.
```

Comment:
As discussed in person we will want to hide the need to join the values together from users in a future slice. I wonder if we could add a copy id button that is specific to the APIs (Or change the existing one in a backwards-compatible way)...

Reply:
Yeah, we should have this information available in the front-end so that should be a quick add.